### PR TITLE
Ironnet: Refactoring - Update Splunk SOAR to Python3

### PR DESCRIPTION
**Pull request by guillermo-ironnet**
## Release Notes

**Version 1.1.0 - Released August 25, 2022**

* Upgrade Ironnet app to python 3
* Changed formats to f strings for performance
* Lint using yapf

### CODE REVIEW

File: NOTICE [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Splunk SOAR IronNet
-Copyright (c) IronNet, 2020
```
Comment (dgopani-crest) [inline NOTICE:2]: This is update so it should be `Copyright (c) IronNet, 2020-2022` in all places

File: ironnet.json [hunk 2]
```diff
@@ -10,11 +10,13 @@
     "product_version_regex": ".*",
     "publisher": "IronNet",
     "license": "Copyright (c) IronNet, 2020",
-    "app_version": "1.0.3",
+    "app_version": "1.1.0",
```
Comment (dgopani-crest) [inline ironnet.json]: 2.0.0

File: release_notes/unreleased.md [hunk 3]
```diff
@@ -1 +1,3 @@
 **Unreleased**
+* Upgrade splunk soar app to python 3
```
Comment (dgopani-crest) [inline release_notes/unreleased.md]: Compatibility changes for Python 3 support

**Review by dgopani-crest — APPROVED** [review]

File: release_notes/unreleased.md (hunk 3 above)
Comment (mishalp-crest) [reply release_notes/unreleased.md]: done

File: NOTICE (hunk 1 above)
Comment (mishalp-crest) [reply NOTICE:2]: done

File: ironnet.json (hunk 2 above)
Comment (mishalp-crest) [reply ironnet.json]: done

**Review by mmomin-crest — APPROVED** [review]

**Review by btavethiya-crest — APPROVED** [review]

**Comment by marcus-defazio** [comment]
@mishalp-crest, is this PR ready to be merged?

**Comment by mishalp-crest** [comment]
> @mishalp-crest, is this PR ready to be merged?

@marcus-defazio Not yet, we need one more approval from the given reviewers

**Review by splunk-jessica — APPROVED** [review]

**Review by paulnawat — APPROVED** [review]

**Pull request merged**